types/v14.go

```diff
@@ -19,6 +20,19 @@ import (
 
 type MetadataV14Decoder struct {
 	ScaleDecoder
+	Errors []MetadataModuleErrorV14 `json:"errors"`
+}
+type MetadataModuleErrorV14 struct {
+	ScaleDecoder `json:"-"`
+	Index        int `json:"index"`
+}
+
+func (m *MetadataModuleErrorV14) Init(data scaleBytes.ScaleBytes, option *ScaleDecoderOption) {
+	m.ScaleDecoder.Init(data, option)
+}
+func (m *MetadataModuleErrorV14) Process() {
+	cm := MetadataModuleErrorV14{}
+	cm.Index = m.ProcessAndUpdateData("U8").(int)
```

Comment:
This method is not needed because the index is included in the metadata v14 Variants and does not need to be parsed again here.